package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "trainee-ui-app",
-  "version": "0.56.6",
+  "version": "0.56.7",
```

Comment:
No need to bump version for README changes

Reply:
version 1.0.0 ?

Reply:
That I could agree with 😁